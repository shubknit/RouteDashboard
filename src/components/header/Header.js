import React from 'react';

// This component is create a header
export const Header = () => (
	<header className ="header">
		<div className ="content-container">
			<div className = 'header__content'>
    			<h1>Router Finder App</h1>
			</div>	
    	</div>
	</header>
)