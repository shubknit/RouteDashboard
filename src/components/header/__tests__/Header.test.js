import React from 'react';
import { Header } from '../Header';
import renderer from 'react-test-renderer';


it('should render the header correctly', () => {
    const wrapper = renderer.create(<Header/>).toJSON();
    expect(wrapper).toMatchSnapshot();
})