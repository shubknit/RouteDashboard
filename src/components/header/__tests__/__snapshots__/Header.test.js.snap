// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header correctly 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <h1>
        Router Finder App
      </h1>
    </div>
  </div>
</header>
`;
